src/libs/ReactNativeLauncher.js

```diff
@@ -26,6 +26,16 @@ class ReactNativeLauncher extends Launcher {
     this.workerListenedEventsNames = ['log', 'workerEvent', 'workerReady']
   }
 
+  setLogger(onKonnectorLog) {
+    this.logger = onKonnectorLog
+  }
+
+  async log(logContent) {
```

Comment:
why this method is async ? Since there is no away in it?

Reply:
Shouldn't !

Reply:
Done